src/styles/index.ts

```diff
@@ -413,6 +413,11 @@ const styles = (theme: ThemeColors) =>
             fontSize: variables.fontSizeExtraSmall,
         },
 
+        textNormalBold: {
```

Comment:
Are you sure we don't have an existing style for this? We use bold text all throughout the app. Let's try to avoid writing styles if there is a style that exists that we can reuse.

Reply:
Actually, I cant find something equivalent in the styles directory.

Reply:
Maybe textStrong. Let me try that

```
textStrong: {
            fontFamily: FontUtils.fontFamily.platform.EXP_NEUE_BOLD,
            fontWeight: FontUtils.fontWeight.bold,
        }
```

Reply:
resolved!